packages/esbuild-plugin/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@honeybadger-io/esbuild-plugin",
+  "version": "1.0.0",
+  "description": "Esbuild plugin to upload source maps to Honeybadger's API - http://docs.honeybadger.io/guides/source-maps.html",
+  "main": "./dist/index.js",
+  "scripts": {
+    "build": "tsc",
```

Comment:
I could've have used esbuild to actually transpile the Typescript code into javascript, but esbuild does not support type declarations, so I went with Typescript's compiler.